packages/app/src/app/components/UploadProgress/Cube/elements.ts

```diff
@@ -9,35 +9,37 @@ const rotateAnimation = keyframes`
   }
 `;
 
-export const Cube = styled.div`
-  position: absolute;
-  width: ${props => props.size + 1}px;
-  height: ${props => props.size + 1}px;
-
-  * {
+export const Container = styled.div`
+  ${({ size }: { size: number }) => css`
```

Comment:
@Saeris I think it's better to extract the `ContainerProps` here, just to be consistent